Apps/Examples/Examples/All Examples/Custom2DPuckExample.swift

```diff
@@ -4,7 +4,9 @@ import MapboxMaps
 @objc(Custom2DPuckExample)
 public class Custom2DPuckExample: UIViewController, ExampleProtocol {
 
+    internal var toggleAccuracyRadiusButton: UIButton?
     internal var mapView: MapView!
+    internal var shouldToggleAccuracyRadius: Bool = true
```

Comment:
It would be more clear to represent the current state ("showsAccuracyRing") and use a property observer to sync the state changes to the view.